core/server/api/v2/members.js

```diff
@@ -0,0 +1,35 @@
+const memberUserObject = require('../../services/members').api.memberUserObject;
+
+const members = {
+    docName: 'members',
+    browse: {
+        options: [
+            'limit',
+            'fields',
+            'filter',
+            'order',
+            'debug',
+            'page'
+        ],
+        permissions: true,
+        validation: {},
+        query(frame) {
+            return memberUserObject.list(frame.options);
+        }
+    },
+
+    read: {
+        headers: {},
+        data: [
+            'id',
+            'email'
+        ],
+        validation: {},
+        permissions: true,
+        query(frame) {
+            return memberUserObject.get(frame.data, frame.options);
```

Comment:
What are you expecting in `frame.options` here? It looks as though it should be empty, unless the pipeline injects defaults in?

Reply:
The options are used in `toJSON` method on model during final serialization, which in this case happens in the service. Since service doesn't has access to `frame` like normally output serializers do, its been passed through here to be used [here](https://github.com/TryGhost/Ghost/pull/10435/files#diff-459fd032db1b611a543ddb5af8d7d3b4R41). The options themselves could be empty or have defaults, this keeps the flow consistent with rest of the `read` flows.

Reply:
Ah, yeah looks its gonna be empty rn, till we add any options to `read`. Pipeline doesn't inject anything here I think